communities/microgalaxy/lab/usegalaxy.eu.yml

```diff
@@ -15,7 +15,7 @@ nationality: European
 # Used for rendering tool/workflow links. Trailing '/' will be removed.
 galaxy_base_url: https://microgalaxy.usegalaxy.eu
 subdomain: microGalaxy
-root_domain: usegalaxy.eu
+root_domain: https://usegalaxy.eu
```

Comment:
This change will break things, e.g. in https://github.com/galaxyproject/galaxy_codex/blob/3605e5c8fd6995bd77ffc25b41330ae6368c695c/communities/microgalaxy/lab/templates/intro.html#L29

Reply:
Probably better to add `https` in the `href` where you use `{ root_domain }` directly